.github/workflows/action_server_binary_release.yml

```diff
@@ -70,4 +75,38 @@ jobs:
         with:
           name: action-server-version
           path: "action_server/version.txt"
-          
+
+      - name: Copy assets [Linux]
+        if: matrix.os == 'ubuntu-20.04'
+        run: |
+            cp -R build-binary/build/x86_64-unknown-linux-gnu/debug/install/* go-wrapper/assets
+      - name: Copy assets [Mac]
+        if: matrix.os == 'macos-11'
+        run: |
+            cp -R build-binary/build/x86_64-apple-darwin/debug/install/* go-wrapper/assets
+      - name: Copy assets [Windows]
+        if: matrix.os == 'windows-2019'
+        run: |
+            xcopy /e /h /i build-binary\\build\\x86_64-pc-windows-msvc\\debug\\install go-wrapper\\assets
+
+      - uses: actions/download-artifact@v4
+        with:
+          name: action-server-version
+          path: "action_server/go-wrapper/assets"
+      
+      - name: Build Go binary wrapper
+        run: |
+            cd go-wrapper
+            go build
+
+      - uses: actions/upload-artifact@v4
+        if: ${{ matrix.os == 'ubuntu-20.04' || matrix.os == 'macos-11'  }}
+        with:
+          name: ${{ runner.os }}-action-server-wrapper
+          path: "action_server/go-wrapper/action-server-wrapper"
+
+      - uses: actions/upload-artifact@v4
+        if: matrix.os == 'windows-2019'
+        with:
+          name: ${{ runner.os }}-action-server-wrapper
+          path: "action_server/go-wrapper/action-server-wrapper.exe"
```

Comment:
The output executable needs to be named `action-server` as that is what users will be writing in CLI.